packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -3513,20 +3513,16 @@ export default class Meeting extends StatelessWebexPlugin {
    * @returns {Promise<void>} resolves when the brb status is updated or does nothing if not in a multistream meeting.
    * @throws {Error} - Throws an error if the request fails.
    */
+  // eslint-disable-next-line consistent-return
```

Comment:
What if you remove the `else`? So something like:
```
public async beRightBack(enabled: boolean): Promise<void> {
    // this logic should be applied only to multistream meetings
    if (!this.isMultistream) {
      LoggerProxy.logger.error('Meeting:index#beRightBack --> Not a multistream meeting');
    } else if (!this.mediaProperties.webrtcMediaConnection) {
      LoggerProxy.logger.error(
        'Meeting:index#beRightBack --> WebRTC media connection is not defined'
      );
    }

    return this.meetingRequest
      .sendBrb({
        enabled,
        locusUrl: this.locusUrl,
        deviceUrl: this.deviceUrl,
        selfId: this.selfId,
      })
      .then(() => {
        this.sendSlotManager.setSourceStateOverride(MediaType.VideoMain, enabled ? 'away' : null);
      })
      .catch((error) => {
        LoggerProxy.logger.error('Meeting:index#beRightBack --> Error ', error);

        return Promise.reject(error);
      });
  }
```

Reply:
@brycetham, this would cause a source state to be always sent. I think that we should not be sending the source state change if the meeting is not multistream or webrtc media is not defined. Or I am wrong?

Reply:
@SomeBody16, seems like that is not the case. Triple checking the `LoggerProxy.logger.error` it seems like it's doing a  `console.error` for the error message. I think we should throw an additional error/have a return statement/or keep the current logic.
<img width="544" alt="Screenshot 2024-11-19 at 10 41 17 AM" src="https://github.com/user-attachments/assets/44c6588f-c0d8-4acb-9fd0-4fd49833c104">

<img width="1365" alt="Screenshot 2024-11-19 at 10 39 49 AM" src="https://github.com/user-attachments/assets/a2a285c1-b7f5-4592-a06c-d404d59bdfb8">
<img width="1369" alt="Screenshot 2024-11-19 at 10 39 58 AM" src="https://github.com/user-attachments/assets/bfbe7523-2bba-4f0d-a204-a194eb3bb526">